pages/app/calendar.js

```diff
@@ -26,44 +26,46 @@ import { data } from '../../components/Calendar/data/dummyData.js';
 let username = 'Matt';
 
 
-export default function Calendar () {
-//import user state (mentor/mentee)
+export default function Calendar() {
+  //import user state (mentor/mentee)
 
   const [appointmentData, setAppointmentData] = useState([]);
 
 
-    useEffect(() => {
-      setAppointmentData(data);
+  useEffect(() => {
+    setAppointmentData(data);
   }, []);
 
   return (
-    <>
-      <Head>
+    <div className="pageData">
+      <div className={styles.calendarContainer}>
+        <Head>
           <title>{`${username}'s Calendar`}</title>
```

Comment:
@gunpowder66 @changerbang  Please review this file to make sure everything looks good.

Reply:
I believe Tiff just added line 40 and 41 "pageData" and "styles.caleendarContainer". If thats just for styling, should be fine.

Reply:
I believe Tiff just added line 40 and 41 "pageData" and "styles.caleendarContainer". If thats just for styling, should be fine.